server/tserver/src/main/java/org/apache/accumulo/tserver/AssignmentHandler.java

```diff
@@ -190,6 +189,7 @@ public void run() {
           server.recentlyUnloadedCache.remove(tablet.getExtent());
         }
       }
+      tablet.setLastAccessTime();
```

Comment:
This may not be needed if this can be set during tablet creation. If it is needed, should probably be done before the tablet is added to online tablets.  Once its added to online tablets it can be considered for unload.

Reply:
Implemented in a7361ea